### DISCUSSION TITLE
CSV Import error [Nov 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4439] [Discord Threads]
> 🙋‍♂️ **Vlas**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Help us by providing additional details on issue observed on GitHub (https://github.com/nocodb/nocodb/issues). If feasible, attach a redacted CSV sample.  
#  
> 💁 **o1lab**  
#### plus DM the csv you have will try it out  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1038341827683680277)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
